cpp/util/masterelection.cc

```diff
@@ -88,6 +99,8 @@ MasterElection::MasterElection(const shared_ptr<libevent::Base>& base,
       backed_proposal_(kNoBacking),
       is_master_(false) {
   CHECK_NE(kNoBacking, node_id);
+  is_master_gauge->Set(0);
+  participating_in_election_gauge->Set(0);
```

Comment:
More than one election in the same process?
...
![crazyfool](https://cloud.githubusercontent.com/assets/7648032/6376395/67cbe63c-bd15-11e4-849c-19a954ce5d0a.jpg)